### PR TITLE
Accordion improvements - design and missing data

### PR DESCRIPTION
**Pull request by omerxx**
Widened accordion, added accident id and provider code, added date to its section

image

### CODE REVIEW

File: static/css/accordion.css [hunk 1]
```diff
@@ -69,7 +69,7 @@
 	z-index: 10;
 	transition: 
 		max-height 0.4s ease-out, 
-		height 0.1s ease-in-out, 
+		height 0.1s ease-in-out,
```
Comment (OmerSchechter) [inline static/css/accordion.css]: What are those changes?

**Comment by OmerSchechter** [comment]
I suggest you check that with @galraij, as I understood from him we are leaning towards making the "accordion" thinner rather than expanding elements to occupy the extra white space.

**Comment by danielhers** [comment]
Waiting for @galraij 's input/

**Comment by omerxx** [comment]
@galraij  ??

**Comment by danielhers** [comment]
@omerxx also need to rebase and resolve conflicts.

**Comment by omerxx** [comment]
Sure, once this is approved I will

**Comment by galraij** [comment]
I would say as thin as possible since we are going to change many of the parameters to icons.

**Comment by omerxx** [comment]
Why would that affect the position and size of lines inside the marker?
I changed it because UI wise this seems wrong to me..

**Comment by galraij** [comment]
But if we move the logo of the Lamas the line that says “source” and make it smaller we save a lot of space, not? The story can spread to more lines...

**Comment by omerxx** [comment]
Of course. That's a decision to make - whether you want to shrink things up or widen the accordion.
Regarding the story - I think the current size is optimal, having it squeezed into more lines would not be comfortable for reading. We could reduce the font size though.

One way to go is making everything thiner or making everything wider, the offered solution here is for the current state - where the marker is still at its original size, and containing an accordion, they should complete each other (design-wise)..

We can continue the discussion tomorrow and make these decisions...

**Pull request closed without merging**